CHANGELOG.md

```diff
@@ -0,0 +1,321 @@
+# Changelog
+
+All notable changes to this project will be documented in this file - formatted and maintained 
+according to the rules documented on http://keepachangelog.com.
```

Comment:
I would recommend to put [urls in angle brackets](https://www.markdownguide.org/basic-syntax/#urls-and-email-addresses) for the case, that the rendering engine doesn't support [Automatic URL Linking](https://www.markdownguide.org/extended-syntax/#automatic-url-linking):
```suggestion
according to the rules documented on <http://keepachangelog.com>.
```